Sources/server/SwiftModulesAdapter.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  SwiftModulesAdapter.swift
+//  swift-index
+//
+//  Created by Cory Alder on 5/12/16.
+//
+//
+
+import shared
+import HTTPSClient
+import Vapor
+import JSON
+
+struct SwiftModulesAdapter: ThirdPartyIndexAdapter {
+    
+    var name: String = "Swift Modules"
+    
+    func search(_ query: String) throws -> [PackageInfo] {
+        
+        let raw = try self.fetch(query)
+        let results = try self.convert(raw)
+        return results
+    }
+    
+    private func fetch(_ query: String) throws -> JSON {
+        //https://swiftpkgs.ng.bluemix.net/api/search/jay?page=1&items=13
```

Comment:
Is this copy pasta from the IBM adapter? Maybe just replace it with your sample search URL :)